### ISSUE TITLE
MovieApp: Detail Page

### ISSUE THREAD

**Issue by RobertBReich** [RobertBReich/capstone-project#4]
# Value Proposition

**As a** user
**I want to** see details of a certain movie/tv-show
**So that** I can decide more easily which movie I want to watch

# Description

![Image](https://user-images.githubusercontent.com/107849620/185155851-a745af2e-aca3-44ca-8737-61623585e443.png)

# Acceptance Criteria

- [x] on click on any shown movie image, the user is directed to the detail page of that movie or tv-show
- [x] when the user clicks on the back button, he gets back to the overview page

# Tasks

- [x] create a detail page
- [x] insert an onclick event that loads the data if necessary or access it from the Store 
- [x] on the detail page, create Components for an Image, a headline and a the description text.
- [x] add a back button to get back to the overview page

# Complexity

M

**Comment by ahohnsen** [comment]
Hey Robert,

Good job 🚀 I have a few suggestions:

Value proposition:
- I would rephrase your value proposition a bit. In the second line you wanna write why the user would want that feature which you wanna implement. In this example: **Why** does the user want to see details of a certain movie/tv-show? I would propose something like:

_As a user
I want to see details of a certain movie/tv-show
so that I can decide more easily which movie I want to watch_

Scribble:
- Please delete all elements which will be implemented later on such as the navigation. As well it is best to fill your scribble with real content instead of placeholders like "movie title".

Acceptance Criteria:
- I suggest to write the criteria from another perspective without using "you". Example:

_on click on any shown movie image, the user is directed to the detail page of that movie or tv-show_

instead of:

> on click on any shown movie image, you are directed to the detail page of that movie or tv-show

- In the acceptance criteria we want to describe all possible user interactions. So here you would also need to write what happens when the user clicks on the back-button. 🙂 
- Everything which is already visible in the scribble does not need to be inside the acceptance criteria (e.g. there is an image, header...)
